package.json

```
@@ -12,10 +12,11 @@
     "prepublishOnly": "npm run swagger",
     "pretest": "npm run swagger",
     "test": "npm run cover && npm run report",
-    "posttest": "npm run eslint",
+    "posttest": "npm run lint",
     "report": "nyc report --reporter=lcov",
-    "eslint": "eslint-godaddy -c .eslintrc lib/ test/",
+    "lint": "eslint-godaddy -c .eslintrc lib/ test/",
```

does this need to be:
```suggestion
    "lint": "eslint-godaddy -c .eslintrc \"{lib,test}/**/*.js\"",
```

Seems like some files were maybe missed?